packages/rollup-plugin/src/index.ts

```diff
@@ -74,8 +74,9 @@ export function vanillaExtractPlugin({
     },
     // Emit .css assets
     moduleParsed(moduleInfo) {
-      moduleInfo.importedIdResolutions.forEach((resolution) => {
-        if (resolution.meta.css) {
+      moduleInfo.importedIds.forEach((id) => {
```

Comment:
currently tying to use `rolldown` gives the following error:

![image](https://github.com/user-attachments/assets/d0562667-d0fd-43d8-beb3-f4950bc24834)

but with this patched version we are able to successfully run the plugin in `rolldown`:

![image](https://github.com/user-attachments/assets/d973ea73-ea38-4bbc-9d61-765e8933afb3)